packages/shell-server/shell-server.js

```diff
@@ -441,7 +441,18 @@ function evalCommand(command, context, filename, callback) {
   }
 
   evalCommandPromise.then(function () {
-    callback(null, script.runInThisContext());
+
+    if (repl.input) {
+      callback(null, script.runInThisContext());
+    } else {
+      // If repl didn't start, `require` and `module` are not visible
+      // in the vm context
+      repl.require = require;
+      repl.module = module;
```

Comment:
And of course now would be a great time to move the code that creates the REPL module into a helper function, since it's now used twice.